### DISCUSSION TITLE
Trouble

### DISCUSSION THREAD

**Discussion by Solo1245** [python-telegram-bot/python-telegram-bot#3831] [Q&A]
Im having trouble with 1 line of code which is stopping me from getting my bot working

updater = Updater(API_TOKEN,True)
dispatcher = updater.dispatcher

it comes up with this 
![image](https://github.com/python-telegram-bot/python-telegram-bot/assets/127560097/f36d343c-d625-4c35-bf9c-aa3937fbe388)
Would really like a reply on how to fix it

**Comment by lemontree210** [comment]
Hey. You seem to be using a version >=20.0 of python-telegram-bot  but your code is written for version <=13.15. Please either upgrade your code to  v20 by reading the [release notes](https://telegra.ph/Release-notes-for-python-telegram-bot-v200a0-05-06) and the [transition guide](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Transition-guide-to-Version-20.0) or install a version of PTB that is compatible with your code base.

Please note that we only provide support for the latest stable version and that the library has undergone significant changes in v20.
